web/apps/labelstudio/src/pages/DataManager/DataManager.jsx

```diff
@@ -225,7 +211,29 @@ export const DataManagerPage = ({ ...props }) => {
       <Button to="/projects">Back to projects</Button>
     </Block>
   ) : (
-    <Block ref={root} name="datamanager" />
+    <>
+      {loading && (
+        <div
+          style={{
+            flex: 1,
+            position: "absolute",
+            top: 0,
+            left: 0,
+            right: 0,
+            bottom: 0,
+            width: "100%",
+            height: "100%",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
```

Comment:
a css class here would be ideal if possible

Reply:
Done